modules/extensions-support/src/main/java/org/mule/runtime/module/extension/internal/runtime/operation/AbstractReturnDelegate.java

```diff
@@ -188,8 +196,10 @@ protected Message toMessage(Object value, ExecutionContextAdapter operationConte
       if (value.getOutput() instanceof InputStream) {
         ConnectionHandler connectionHandler = (ConnectionHandler) operationContext.getVariable(CONNECTION_PARAM);
         if (connectionHandler != null && supportsStreaming(operationContext.getComponentModel())) {
-          value = value.copy()
-              .output(new ConnectedInputStreamWrapper((InputStream) value.getOutput(), connectionHandler))
+          ConfigurationInstance config = (ConfigurationInstance) operationContext.getConfiguration().get();
```

Comment:
Do not call the `get` method without checking if the optional is present or not.